src/select/interfaces.ts

```diff
@@ -92,6 +92,11 @@ export interface BaseSelectProps
    * @i18n
    */
   filteringClearAriaLabel?: string;
+  /**
```

Comment:
move these to `SelectProps` below rather than `BaseSelectProps` so they don't get inherited by `Multiselect`